vue-frontend/src/components/home/HomeHeader.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <nav class="navbar navbar-expand-md navbar-light main-header" id="mainHeader">
+    <div class="container">
+      <div class="navbar-brand" onclick="window.location.replace('/')">
+        <img :src="headerImage" class="header-logo-image" alt="logo" />
+      </div>
+      <div class="navbar-collapse">
+        <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link normal-text" :href="careerURL" target="_blank"
+              >Career</a
+            >
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link blogs normal-text"
+              :href="blogsURL"
+              target="_blank"
+            >
+              Blogs</a
+            >
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link start-btn normal-text gradient-border-btn"
+              :href="getStartedURL"
+              target="_self"
+              ><span
+                ><span class="canopas-gradient-text">Get started &nbsp;</span>
+                <font-awesome-icon class="faArrowRight" :icon="faArrowRight" />
+              </span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link start-btn-link normal-text"
+              :href="getStartedURL"
+              target="_self"
+            >
+              Get started</a
+            >
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</template>
+
+<script type="module">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import headerImage from "@/assets/images/logo/logo-header.svg";
+
+library.add(faArrowRight);
```

Comment:
What if we add all required icons in the library and place this code in `Home.vue` instead of all components?

Reply:
It's fine to have it in component. Icon will be imported only once.

Reply:
@cp-shruti-s Can you revert the change? I think we will need to use this header outside `HomeView` as nidhi mentioned.

Reply:
okay but revert all MR change yaa just HomeHeader file change ?? @jimmy0251